libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -1,18 +1,41 @@
+#include <string.h>
+#include <stdbool.h>
```

Comment:
I think we should avoid stdbool.h

Reply:
I mistakenly removed stdbool.h from this file in #959, however it's required in the unit tests to avoid build warnings.

Reply:
Why would we require it if we use the same technique as in mqtt_light_weight.h here? I think if we target library to be C89, the unit tests also should be targeting the same.

Reply:
CMock autogenerates several files in the build directory to be used in mocks. Some of the files are including stdbool.h, so if we don't include it here as well, it will result in build warnings

Reply:
It appears that stdbool.h is automatically added to mock files due to this [file](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/tools/cmock/project.yml#L22). However, removing that line results in build errors due to other tests in [http](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/libraries/standard/http/utest/http_utest.c#L279). Removing stdbool.h seems to be a change wider than this PR.

Reply:
Ok. I think it can be done as a different PR then.